test/topology_custom/test_raft_ignore_nodes.py

```diff
@@ -4,27 +4,71 @@
 # SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
 #
 import time
+from typing import Optional, List, Any
 import pytest
 import logging
 
 from test.pylib.internal_types import IPAddress, HostID
 from test.pylib.scylla_cluster import ReplaceConfig
-from test.pylib.manager_client import ManagerClient
-from test.topology.util import wait_for_token_ring_and_group0_consistency
+from test.pylib.manager_client import ManagerClient, ServerInfo
+from test.topology.util import get_current_group0_config, wait_for_token_ring_and_group0_consistency
 
 
 logger = logging.getLogger(__name__)
 
 
+async def make_servers(manager: ManagerClient, servers_num: int,
+                       config: Optional[dict[str, Any]] = None) -> List[ServerInfo]:
```

Comment:
I honestly wouldn't mind relying on the implementation a bit more to make it simpler. We could assume that voters won't change after we add 5 nodes. Then you could basically do:
```python
voters = await manager.servers_add(5, config=config)
non_voters = await manager.servers_add(servers_num - 5, config=config)
```
But I am also fine with what you did because it doesn't really complicate the test. If it fails and someone has to investigate it, they don't even have to go into details of `make_servers`, they can just read the new comment and continue the investigation.

Reply:
We would also have to assume that there are 5 voters by default. So maybe it's not a good idea.